src/modules/jobs/types/job-data.type.ts

```diff
@@ -0,0 +1,10 @@
+export interface JobData {
+  classData: {
+    retries: number
+    priority: number
+    createdAt: number
+    queueName: string
+    customerUuid: string
+  }
+  className: string
+}
```

Comment:
is het nodig om `retries`, `priority`, `createdAt` en `queueName` bij te houden? Dit zijn al kolommen op de Job tabel, dus lijkt mij duplication.

en waarom `customerUuid`? Toch niet elke job heeft dit?

```suggestion
export interface JobData {
  classData: unknown
  className: string
}
```

Reply:
dat lijkt me niet juist, eerder zoals maarten zegt